src/rules/font_feature_values.rs

```diff
@@ -0,0 +1,128 @@
+//! The `@font-feature-values` rule.
+
+use cssparser::*;
+
+use super::Location;
+use crate::declaration::DeclarationBlock;
+use crate::error::{ParserError, PrinterError};
+use crate::parser::DefaultAtRule;
+use crate::printer::Printer;
+use crate::rules::CssRuleList;
+use crate::traits::{Parse, ToCss};
+use crate::values::ident::CustomIdent;
+use crate::values::string::CowArcStr;
+#[cfg(feature = "visitor")]
+use crate::visitor::Visit;
+
+/// A [@font-feature-values](https://drafts.csswg.org/css-fonts/#font-feature-values) rule.
+#[derive(Debug, PartialEq, Clone)]
+#[cfg_attr(feature = "visitor", derive(Visit))]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "jsonschema", derive(schemars::JsonSchema))]
+#[cfg_attr(feature = "into_owned", derive(static_self::IntoOwned))]
+pub struct FontFeatureValuesRule<'i, R = DefaultAtRule> {
+  /// The name of the font feature values.
+  pub name: Vec<FontFeatureValuesRuleName<'i>>,
+  /// The declarations within the `@font-feature-values` rule.
+  #[cfg_attr(feature = "serde", serde(borrow))]
+  pub declarations: DeclarationBlock<'i>,
```

Comment:
As far as I can tell, the only declaration supported in the `@font-feature-values` rule is `font-display`. This should ideally work more like `@font-face` and `@font-palette-values`, where there is a special property type for the values of this at rule rather than the generic DeclarationBlock which applies only to style rules.

Reply:
CSSWG recommends removing font-display from this at-rule. https://github.com/w3c/csswg-drafts/issues/2973#issuecomment-2418922905